docs/source/release-notes/2.1.0.rst

```diff
@@ -5,3 +5,8 @@ Features Added
 ``````````````
 - Add ``maintainer_can_modify`` parameter to ``create_pull`` method of
   ``Repo`` class.
+
+CI/CD
+`````
+
+- Remove deprecated ``setup.py check`` in favor to ``twine check``.
```

Comment:
I doubt anyone cares about this. The release notes should be user-focused and this isn't a user-focused change

Reply:
You are right, I reverted it.